backend/src/main/kotlin/com/hiper2d/auth/token/GuestAuthenticationToken.kt

```diff
@@ -0,0 +1,18 @@
+package com.hiper2d.auth.token
+
+import org.springframework.security.authentication.AbstractAuthenticationToken
+import org.springframework.security.core.GrantedAuthority
+
+class GuestAuthenticationToken(
+    authorities: List<GrantedAuthority> = emptyList(),
```

Comment:
it's like one time token till you refresh or leave page? Do we need it on prod?

Reply:
The idea is to have a token in every request. It can be saved to a local storage/cookies. It this case even unauthorized user will be able to keep something in his local storage. I haven't yet finished it.  It's a draft PR. Don't know if saving a token in local storage is a good practice.